src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -739,9 +771,11 @@ function build_eal_list() {
 ))->setHelp('Select the interface for the local endpoint of this phase1 entry.');
 
 if (!$pconfig['mobile']) {
-	$section->addInput(new Form_Input(
+	$group = new Form_Group('*Remote Gateway');
+
+	$group->add(new Form_Input(
 		'remotegw',
-		'*Remote Gateway',
+		'Remote Gateway',
```

Comment:
Remote Gateway is a mandatory config item, please add `*` back

Reply:
This looks OK, the `*` is now on the group (Right side, line 774), not just this one option.

Reply:
Oh, right.  I've missed that